app/views/works/homepage.html.erb

```diff
@@ -0,0 +1,7 @@
+Media Spotlight: <%= @ranked_works.first.title %> by <%= @ranked_works.first.created_by %> <br>
+<%#= @ranked_works.first.votes %> - <%= @ranked_works.first.description %> <%# will be order by vote count next %> <br><br>
+<p>Top Movies: <%= @ranked_works.where(category: "movie")[0..10] %></p>
```

Comment:
Mentioned this in a different file, but it may be better to move the logic that defines this set of works in the controller action, like `@top_movies = @ranked_works.where(category: "movie")[0..10]`. We'll want to keep complex logic like this out of the view